tests/training/test_loss_functions.py

```diff
@@ -6,7 +6,10 @@
 from torch import Tensor
 from torch.autograd import grad
 
-from graphnet.training.loss_functions import LogCoshLoss, VonMisesFisherLoss
+from graphnet.training.loss_functions import (
+    LogCoshLoss,
+    VonMisesFisherLoss,
+)
```

Comment:
Change is not pertinent to the goal of the PR.
```suggestion
from graphnet.training.loss_functions import LogCoshLoss, VonMisesFisherLoss
```

Reply:
This is actually something black did at some point. How do I apply black only to my own code?